src/components/Squads/SquadCard.tsx

```diff
@@ -82,7 +82,7 @@ export default function SquadCard({squad, classDescriptions}) {
                 <Box>
                     <GamerLinkList gamers={gamerList}/>
 
-                    <ClassBadgeList subject={squad as object}
```

Comment:
using "object" is a typescript no no, that's why I use Record<any, unknown> to represent an object (as recommended by eslint & typescript docs)